regression-test/suites/external_table_p0/es/test_es_query.groovy

```diff
@@ -158,6 +158,7 @@ suite("test_es_query", "p0,external,es,external_docker,external_docker_es") {
         order_qt_sql77 """select * from test1 where esquery(test2, '{"match":{"test2":"text#1"}}')"""
         order_qt_sql78 """select c_bool, c_byte, c_short, c_integer, c_long, c_unsigned_long, c_float, c_half_float, c_double, c_scaled_float, c_date, c_datetime, c_keyword, c_text, c_ip, c_person from test1"""
         order_qt_sql79 """select c_bool, c_byte, c_short, c_integer, c_long, c_unsigned_long, c_float, c_half_float, c_double, c_scaled_float, c_date, c_datetime, c_keyword, c_text, c_ip, c_person from test2"""
+        order_qt_sql710 """select * from test3_20231005"""
```

Comment:
This test should be in ES 8.x

Reply:
> This test should be in ES 8.x

What's the meaning? Does ES8 need to be included?

Reply:
Your bad case is in ES 8.9.1, so the test case should be included in 8.x

Reply:
> Your bad case is in ES 8.9.1, so the test case should be included in 8.x

You are right. In fact, es7/8 will have such problems. I will add test cases for es8. thank you for viewing.